plugin/src/main/resources/io/jenkins/plugins/casc/assets/prism.css

```diff
@@ -1,5 +1,23 @@
+/* Custom Jenkins bits to provide a good user experience
```

Comment:
could this go in a different css file? this file is completely generated currently from the prism website

Reply:
Sure not a problem :)